README.md

```diff
@@ -9,6 +9,115 @@
 
 
 
+## MotionAnimator: a drop-in replacement
+
+UIView's implicit animation APIs are also available on the MotionAnimator:
+
+```swift
+// Animating implicitly with UIView APIs
+UIView.animate(withDuration: 1.0, animations: {
+  view.alpha = 0.5
+})
+
+// Equivalent MotionAnimator API
+MotionAnimator.animate(withDuration: 1.0, animations: {
+  view.alpha = 0.5
+})
+```
+
+But the MotionAnimator allows you to animate more properties — and on more iOS versions:
+
+```swift
+UIView.animate(withDuration: 1.0, animations: {
+  view.layer.cornerRadius = 10 // Only works on iOS 11 and up
+})
+
+MotionAnimator.animate(withDuration: 1.0, animations: {
+  view.layer.cornerRadius = 10 // Works on iOS 8 and up
+})
+```
+
+MotionAnimator makes use of the [MotionInterchange](https://github.com/material-motion/motion-interchange-objc), a standardized format for representing animation traits. This makes it possible to tweak the traits of an animation without rewriting the code that ultimately creates the animation, useful for building tweaking tools and making motion "stylesheets".
+
+```swift
+// Want to change a trait of your animation? You'll need to use a different function altogether
+// to do so:
+UIView.animate(withDuration: 1.0, animations: {
+  view.alpha = 0.5
+})
+UIView.animate(withDuration: 1.0, delay: 0.5, options: [], animations: {
+  view.alpha = 0.5
+}, completion: nil)
+
+// But with the MotionInterchange, you can create and manipulate the traits of an animation
+// separately from its execution.
+let traits = MDMAnimationTraits(duration: 1.0)
+traits.delay = 0.5
+
+let animator = MotionAnimator()
+animator.animate(with: traits, animations: {
+  view.alpha = 0.5
+})
+```
+
+The MotionAnimator can also be used to replace explicit Core Animation code with additive explicit animations:
+
+```swift
+// Animating expicitly with Core Animation APIs
+let animation = CABasicAnimation(keyPath: "cornerRadius")
+animation.fromValue = 0
+animation.toValue = 10
+animation.duration = 1.0
+// Note: this animation won't be additive unless we configure it accordingly.
```

Comment:
Where did you configure it to be additive?

Reply:
Hm - I'd intentionally not demonstrated how to implement an additive animation here, but perhaps it's more valuable to have the animation output be equivalent between these two examples? What do you think?

Reply:
I think it's fair to show how to make it additive in CA.  I'm curious how easy/hard it is.

Reply:
Added!